examples/webvr_cubes.html

```diff
@@ -26,7 +26,7 @@
 			}
 
 			.button.enabled {
```

Comment:
I guess this class was never getting used so I [handled it below](https://github.com/mrdoob/three.js/pull/8332/files#diff-b860e4efd3f299bea60d5ea2f72c89d8R137)